src/runtime_src/xdp/profile/plugin/ocl/ocl_profiler.cpp

```diff
@@ -689,8 +693,11 @@ namespace xdp {
               numTraceBytes += readBytes;
               info->mTraceVector = {};
             }
-            if (numTraceBytes >= mDDRBufferSz)
+            if (numTraceBytes >= mDDRBufferSz) {
```

Comment:
Please change variable name from mDDRBufferSz to mDDRBufferSize. Easier to read.